crates/e2e/src/xts.rs

```diff
@@ -435,7 +437,8 @@ where
                     .cloned()
                     .enumerate()
                     .find_map(|(index, ext)| {
-                        let hash_ext = Transaction::<C>::from_bytes(ext.0).hash();
+                        let hash_ext = Transaction::<C>::from_bytes(ext.0)
+                            .hash_with(self.client.hasher());
```

Comment:
In line with hashing rework in https://github.com/paritytech/subxt/commit/21b3f521917871771ba6e1b71b70fb53938e87d5, uses the hasher instance available from the client.